docs/content/en/docs/concepts/tracing-policy/selectors.md

```diff
@@ -1537,6 +1537,7 @@ There are different types supported for each operator. In case of `matchArgs`:
 * Protocol
 * Family
 * State
+* Capability (see an [example](https://github.com/cilium/tetragon/blob/main/examples/tracingpolicy/deny-privileged-containers.yml)) - Available for kernel >= 5.4
```

Comment:
I think it deserves at least simple paragraph below as we have for '* State' and other bullets